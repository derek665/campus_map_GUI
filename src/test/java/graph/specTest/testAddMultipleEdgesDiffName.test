# Tests that a graph can be created, a pair of nodes and
# multiple same direction edges can be added, and a pair of nodes
# and the edges can be printed out.

# Create a graph
CreateGraph g5

# Add a pair of nodes
AddNode g5 n1
AddNode g5 n2

# Add same direction edges
AddEdge g5 n1 n2 e1
AddEdge g5 n1 n2 e2
AddEdge g5 n1 n2 e3

# Print nodes in graph and children of n1
ListNodes g5
ListChildren g5 n1