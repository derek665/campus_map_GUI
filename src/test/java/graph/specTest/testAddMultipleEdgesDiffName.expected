# Tests that a graph can be created, a pair of nodes and
# multiple same direction edges can be added, and a pair of nodes
# and the edges can be printed out.

# Create a graph
created graph g5

# Add a pair of nodes
added node n1 to g5
added node n2 to g5

# Add same direction edges
added edge e1 from n1 to n2 in g5
added edge e2 from n1 to n2 in g5
added edge e3 from n1 to n2 in g5

# Print nodes in graph and children of n1
g5 contains: n1 n2
the children of n1 in g5 are: n2(e1) n2(e2) n2(e3)